From efb969b8a86868e7a5507fa0099d6b707e4070e4 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 05 May 2019 18:17:14 +0300
Subject: [PATCH] [SW-0] gaudi: pll: lower the indentation level

There is no need for 2 level indentation in the array items.
Lover the indentation to 1 level.
No functional changes.

Change-Id: I011a396744388f6e486674bafc7d90a233b54477
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index 6e62a98..bb7756d 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -14,93 +14,93 @@
 #include <uart.h>
 
 static struct hl_pll_cfg gaudi_pll_cfg[] = {
-		{
-			.name = "CPU", .base = mmPSOC_CPU_PLL_BASE,
-	//		.nr = 0, .nf = 47, .od = 1, .nb = 23,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 47, -1 }
-		}, {
-			.name = "DMA_W", .base = mmDMA_W_PLL_BASE,
-	//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, -1, -1 }
-		}, {
-			.name = "IF_W", .base = mmIF_W_PLL_BASE,
-	//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "IF_E", .base = mmIF_E_PLL_BASE,
-	//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "DMA_E", .base = mmDMA_E_PLL_BASE,
-	//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, -1, -1 }
-		}, {
-			.name = "MESH_W", .base = mmMESH_W_PLL_BASE,
-	//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "SRAM_W", .base = mmSRAM_W_PLL_BASE,
-	//		.nr = 0, .nf = 35, .od = 1, .nb = 17,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, -1, -1, -1 }
-		}, {
-			.name = "MESH_E", .base = mmMESH_E_PLL_BASE,
-	//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "SRAM_E", .base = mmSRAM_E_PLL_BASE,
-	//		.nr = 0, .nf = 35, .od = 1, .nb = 17,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, -1, -1, -1 }
-		}, {
-			.name = "MME", .base = mmPSOC_MME_PLL_BASE,
-	//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "PCI", .base = mmPSOC_PCI_PLL_BASE,
-	//		.nr = 0, .nf = 37, .od = 1, .nb = 18,
-			.nr = 4, .nf = 331, .od = 3, .nb = 165, //DAY 1 (830MHz)
-			.div_factor = { 0, 0, 4, 9 }
-		}, {
-			.name = "TPC", .base = mmPSOC_TPC_PLL_BASE,
-	//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "HBM", .base = mmPSOC_HBM_PLL_BASE,
-			.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
-			.div_factor = { 0, 1, -1, -1 }
-		}, {
-			.name = "NIC0", .base = mmNIC0_PLL_BASE,
-			.nr = 24, .nf = 1665, .od = 3, .nb = 832, /* 833 MHz */
-			.div_factor = { 0, 4, 8, -1 }
-		}, {
-			.name = "NIC1", .base = mmNIC1_PLL_BASE,
-			.nr = 24, .nf = 1665, .od = 3, .nb = 832, /* 833 MHz */
-			.div_factor = { 0, 4, 8, -1 }
-		}, {
-			.name = "NIC2_HBM", .base = mmNIC2_HBM_PLL_BASE,
-			.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
-			.div_factor = { 0, 1, -1, -1 }
-		}, {
-			.name = "NIC2_MME", .base = mmNIC2_MME_PLL_BASE,
-	//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		}, {
-			.name = "NIC2_TPC", .base = mmNIC2_TPC_PLL_BASE,
-	//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
-			.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
-			.div_factor = { 0, 1, 2, 7 }
-		},
+	{
+		.name = "CPU", .base = mmPSOC_CPU_PLL_BASE,
+//		.nr = 0, .nf = 47, .od = 1, .nb = 23,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 47, -1 }
+	}, {
+		.name = "DMA_W", .base = mmDMA_W_PLL_BASE,
+//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, -1, -1 }
+	}, {
+		.name = "IF_W", .base = mmIF_W_PLL_BASE,
+//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "IF_E", .base = mmIF_E_PLL_BASE,
+//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "DMA_E", .base = mmDMA_E_PLL_BASE,
+//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, -1, -1 }
+	}, {
+		.name = "MESH_W", .base = mmMESH_W_PLL_BASE,
+//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "SRAM_W", .base = mmSRAM_W_PLL_BASE,
+//		.nr = 0, .nf = 35, .od = 1, .nb = 17,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, -1, -1, -1 }
+	}, {
+		.name = "MESH_E", .base = mmMESH_E_PLL_BASE,
+//		.nr = 0, .nf = 59, .od = 1, .nb = 29,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "SRAM_E", .base = mmSRAM_E_PLL_BASE,
+//		.nr = 0, .nf = 35, .od = 1, .nb = 17,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, -1, -1, -1 }
+	}, {
+		.name = "MME", .base = mmPSOC_MME_PLL_BASE,
+//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "PCI", .base = mmPSOC_PCI_PLL_BASE,
+//		.nr = 0, .nf = 37, .od = 1, .nb = 18,
+		.nr = 4, .nf = 331, .od = 3, .nb = 165, //DAY 1 (830MHz)
+		.div_factor = { 0, 0, 4, 9 }
+	}, {
+		.name = "TPC", .base = mmPSOC_TPC_PLL_BASE,
+//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "HBM", .base = mmPSOC_HBM_PLL_BASE,
+		.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
+		.div_factor = { 0, 1, -1, -1 }
+	}, {
+		.name = "NIC0", .base = mmNIC0_PLL_BASE,
+		.nr = 24, .nf = 1665, .od = 3, .nb = 832, /* 833 MHz */
+		.div_factor = { 0, 4, 8, -1 }
+	}, {
+		.name = "NIC1", .base = mmNIC1_PLL_BASE,
+		.nr = 24, .nf = 1665, .od = 3, .nb = 832, /* 833 MHz */
+		.div_factor = { 0, 4, 8, -1 }
+	}, {
+		.name = "NIC2_HBM", .base = mmNIC2_HBM_PLL_BASE,
+		.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
+		.div_factor = { 0, 1, -1, -1 }
+	}, {
+		.name = "NIC2_MME", .base = mmNIC2_MME_PLL_BASE,
+//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	}, {
+		.name = "NIC2_TPC", .base = mmNIC2_TPC_PLL_BASE,
+//		.nr = 0, .nf = 63, .od = 1, .nb = 31,
+		.nr = 0, .nf = 31, .od = 15, .nb = 15, //DAY 1 (100 MHz)
+		.div_factor = { 0, 1, 2, 7 }
+	},
 };
 
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
